src/platform_impl/windows/window.rs

```diff
@@ -487,21 +473,22 @@ impl Window {
                 winuser::PeekMessageW(&mut msg, ptr::null_mut(), 0, 0, 0);
             }
 
-            // Update window style
-            WindowState::set_window_flags(window_state_lock, window.0, |f| {
```

Comment:
I think it would be ideal to keep this part here and adopt the `set_window_flags` implementation by basically moving all the parts below into it. So we change the window style only in a single place and it's also the only usage of the `FULLSCREEN` window flags